docs/source/tutorial_notebooks/getting_started_1_register_datasets.ipynb

```diff
@@ -191,13 +192,11 @@
    "source": [
     "This will register a new dataset. A few notes:\n",
     "\n",
-    "### The relative path\n",
+    "### The dataset name (mandatory)\n",
     "\n",
-    "Datasets are registered within the registry under a path relative to the root directory (`root_dir`), which, by default, is a shared space at NERSC. For those interested, the eventual full path for the dataset will be `<root_dir>/<schema>/<owner_type>/<owner>/<relative_path>`. This means that the combination of `relative_path`, `owner` and `owner_type` must be unique within the registry, and therefore cannot already be taken when you register a new dataset (an exception to this is if you allow your datasets to be overwritable, see below). \n",
+    "The first of two mandatory arguments to the `register()` function is the dataset `name`, which in our example is `nersc_tutorial:my_desc_dataset` (note there is nothing special about the `:` here, the `name` can be any legal string). This should be any convenient, evocative name for the human. Note the combination of `name`, `version` and `version_suffix` must be unique in the database. The dataset `name` allows for an easy retrieval of the dataset for querying and updating.\n",
```

Comment:
They are now illegal

Reply:
I don't see any exclusion of white space characters. Can you add them, or at least blank, to the list?   It would be good to also exclude tab, carriage return and linefeed if possible.